Visualization/src/FCCEventHandler.h

```diff
@@ -0,0 +1,55 @@
+#ifndef DD4HEP_DDEVE_FCCEVENTHANDLER_H
+#define DD4HEP_DDEVE_FCCEVENTHANDLER_H
+
+// Framework include files
+#include "DDEve/EventHandler.h"
+
+#include "podio/EventStore.h"
+#include "podio/ROOTReader.h"
+
+#include <map>
+#include <string>
+
+namespace DD4hep {
+
+class FCCEventHandler : public EventHandler {
+protected:
```

Comment:
make it private. I don't think we want something to inherit from this class, no?